Music.py

```diff
@@ -412,13 +412,16 @@ def randomize_music(rom, settings, music_mapping):
 
     # If not creating patch file, shuffle audio sequences. Otherwise, shuffle pointer table
     # If generating from patch, also do a version check to make sure custom sequences are supported.
-    custom_sequences_enabled = settings.compress_rom != 'Patch'
+    custom_sequences_enabled = not settings.disable_custom_music
     if settings.patch_file != '':
         rom_version_bytes = rom.read_bytes(0x35, 3)
         rom_version = f'{rom_version_bytes[0]}.{rom_version_bytes[1]}.{rom_version_bytes[2]}'
         if compare_version(rom_version, '4.11.13') < 0:
             errors.append("Custom music is not supported by this patch version. Only randomizing vanilla music.")
             custom_sequences_enabled = False
+    elif not custom_sequences_enabled and (settings.background_music == 'random_custom_only' or settings.fanfares == 'random_custom_only'):
+        errors.append("Custom music is disabled when creating patch files. Only randomizing vanilla music.")
+
     if custom_sequences_enabled:
```

Comment:
Apologies if I'm being dumb, but wouldn't this situation be possible?

Generating a patch file and `disable_custom_music` equals False (so custom music is enabled)

`custom_sequences_enabled` = True
 
```           
      v Not True = False, so the custom music isn't caught when it should be
elif not custom_sequences_enabled and (settings.background_music == 'random_custom_only' or settings.fanfares == 'random_custom_only'):
```

It's a little confusing with all the true/false flip flopping but I think that's right